From 0c394ee939a49340e658613acf28a4a64ca694e3 Mon Sep 17 00:00:00 2001
From: Karthik Kambatla <kasha@cloudera.com>
Date: Wed, 23 Jan 2013 15:51:14 -0800
Subject: [PATCH 010/596] MR1: CLOUDERA-BUILD. Backport MAPREDUCE-4562 to MR1
 for compatibility with MR2

Reason: Compatibility with MR2
Ref: CDH-9978
Author: Jarcec
(cherry picked from commit 2c3417107f29a02f0027ec02a53663d282112808)
(cherry picked from commit 6e88f735f18dab4711d4eea04f1ef7d835feeccc)
(cherry picked from commit 936088898c17ab79f54f88be7d2d777ff5ffa2a7)
(cherry picked from commit e923d590e247cc257fc742b21e0b98b6449830c4)
---
 .../mapreduce/counters/AbstractCounters.java       |    2 +-
 .../org/apache/hadoop/mapred/TestCounters.java     |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/hadoop-mapreduce1-project/src/mapred/org/apache/hadoop/mapreduce/counters/AbstractCounters.java b/hadoop-mapreduce1-project/src/mapred/org/apache/hadoop/mapreduce/counters/AbstractCounters.java
index b79a7ca..b5d0cfd 100644
--- a/hadoop-mapreduce1-project/src/mapred/org/apache/hadoop/mapreduce/counters/AbstractCounters.java
+++ b/hadoop-mapreduce1-project/src/mapred/org/apache/hadoop/mapreduce/counters/AbstractCounters.java
@@ -81,7 +81,7 @@
                   TaskCounter.class.getName());
     legacyMap.put("org.apache.hadoop.mapred.JobInProgress$Counter",
                   JobCounter.class.getName());
-    legacyMap.put("FileSystemCounter", FileSystemCounter.class.getName());
+    legacyMap.put("FileSystemCounters", FileSystemCounter.class.getName());
   }
 
   private final Limits limits = new Limits();
diff --git a/hadoop-mapreduce1-project/src/test/org/apache/hadoop/mapred/TestCounters.java b/hadoop-mapreduce1-project/src/test/org/apache/hadoop/mapred/TestCounters.java
index 0b00e41..ab40002 100644
--- a/hadoop-mapreduce1-project/src/test/org/apache/hadoop/mapred/TestCounters.java
+++ b/hadoop-mapreduce1-project/src/test/org/apache/hadoop/mapred/TestCounters.java
@@ -155,7 +155,7 @@ public void testLegacyNames() {
     assertEquals("New name and method", 1, counters.findCounter("file",
         FileSystemCounter.BYTES_READ).getValue());
     assertEquals("Legacy name", 1, counters.findCounter(
-        "FileSystemCounter",
+"FileSystemCounters",
         "FILE_BYTES_READ").getValue());
   }
   
-- 
1.7.9.5

